content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -30,118 +30,122 @@ bare metal and the clusters.
 
     {{< text bash >}}
-    $ echo "${GWIP} istiod.istio-system.svc" | sudo tee -a /etc/hosts
+$ echo "ISTIO_META_NETWORK=vm-network" >> sidecar.env
     {{< /text >}}
 
-   A better options is to configure the DNS resolver of the VM to resolve the address, using a split-DNS server. Using
-   /etc/hosts is an easy to use example. It is also possible to use a real DNS and certificate for Istiod, this is beyond
-   the scope of this document.
-
-1.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs/`.
-
-    {{< text bash >}}
-    $ sudo mkdir -p /etc/certs
-    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
+#### Create Gateway for application traffic
+
+The last step is to create a `Gateway` resource that allows application traffic from the VMs to route correctly.
+
+    {{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: cluster-aware-gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 443
+      name: tls
+      protocol: TLS
+    tls:
+      mode: AUTO_PASSTHROUGH
+    hosts:
+    - "*.local"
     {{< /text >}}
 
-1.  Install `root-cert.pem` under `/var/run/secrets/istio/`.
+Applying this gateway will route any of the traffic from the VM destined for the workloads in the mesh running on `*.local`
 
-1.  Install `cluster.env` under `/var/lib/istio/envoy/`.
+At this point, you can continue with the [Setup Virtual Machine documentation](/docs/setup/install/virtual-machine/).
 
-    {{< text bash >}}
-    $ sudo cp cluster.env /var/lib/istio/envoy
-    {{< /text >}}
+## Verify setup
 
-1.  Transfer ownership of the files in `/etc/certs/` , `/var/lib/istio/envoy/` and `/var/run/secrets/istio/`to the Istio proxy.
+After setup, the machine can access services running in the Kubernetes cluster
+or on other VMs. When a service on the VM tries to access a service in the mesh running on Kubernetes, the endpoints (i.e., IPs) for those services will be the ingress gateway on the Kubernetes Cluster. To verify that, on the VM run the following command (assuming you have a service named `httpbin` on the Kubernetes cluster):
 
     {{< text bash >}}
-    $ sudo chown -R istio-proxy /etc/certs /var/lib/istio/envoy /var/run/secrets/istio/
+$ curl -v localhost:15000/clusters | grep httpbin
     {{< /text >}}
 
-1.  Start Istio using `systemctl`.
+This should show endpoints for `httpbin` that point to the ingress gateway similar to this:
 
-    {{< text bash >}}
-    $ sudo systemctl start istio
+    {{< text text >}}
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_active::1
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_connect_fail::0
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_total::1
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::rq_active::0
```

Comment:
Hm, had to change `gateways` to

```
          gateways:
          - ip: 172.18.0.3
            port: 31798
```

because I don't have a `LoadBalancer` running.

But I still only see the backend service's `PodIP` here, rather than the gateway IP/port.